core-java/src/main/java/org/spine3/server/EntityRepository.java

```diff
@@ -41,41 +45,53 @@
 
     @Nullable
     @Override
-    protected EntityStorage<I, M> getStorage() {
```

Comment:
What happens if we pass an EntityRecord with the state, which is not of the same type with those already stored in the repository?